js/pmpro-login.js

```diff
@@ -12,7 +12,7 @@ jQuery(document).ready(function(){
 		
 		var strength;		
 		if ( pass1 != '' ) {
-			strength = wp.passwordStrength.meter( pass1, wp.passwordStrength.userInputBlacklist(), pass1 );
+			strength = wp.passwordStrength.meter( pass1, wp.passwordStrength.userInputDisallowedList(), pass1 );

```

Comment:
```suggestion
			// Call the disallowed list method corresponding to appropriate WP version.
			const disallowedList = ( 'function' == typeof wp.passwordStrength.userInputDisallowedList )
				? wp.passwordStrength.userInputDisallowedList()
				: wp.passwordStrength. userInputBlacklist();

			strength = wp.passwordStrength.meter( pass1, disallowedList, pass1 );

```

Reply:
@ideadude What about this?

Reply:
At first I thought something like this "doesn't make sense"... adding too much tech debt and kind of highlighting the terms we want to avoid... but it's growing on me as I look at it. It works. I like it.